### ISSUE TITLE
Only list 5 of my repositories. I provided one that it missed

### ISSUE THREAD

**Issue by rogerjdeangelis** [JoshMcguigan/traffic#8]
Nice simple script for a difficult problem                                                         
                                                                                                   
I don't seem to get a list of all my repositories that have traffic   
                             
https://github.com/rogerjdeangelis                 
                                                                                  
I have over 1,000 repositories and have averaged over 100 views per day .                           
My previous API {python}\Lib\site-packages\github_traffic_stats2 no longer works.                   
                                                                                                                                                                                                     
c:\>C:\Users\beast\.cargo\bin\traffic.exe                                                          
                                                                                                   
Repository Name                               Unique Visits         Trend                          
                                              (last 14 days)                                       
                                                                                                   
utl-reducing-two-table-join-from-hours-to-under-three-minutes-SQL-index            4    None       
utl-python-base64-encode-and-decode-a-binary-execl-workbook-or-binary-file         4    None       
utl-populate-or-operate-on-each-element-of-datastep-temporary-or-permanent-array   4    None       
utl-r-iml-and-datastep-sorting-one-million-randomly-ordered-eight-byte-floats      3    Up 1 Day   
utl-removing-the-suffix-in-firm-names-like-LLP-CORP                                1    None       
                                                                                                   
                                                                                                   
Jus one repo not in the report                                                                     
Popular content                                                                           Unique   
                                                                                  Views   Visitors 
Content                                                                                            
utl-adding-images-to-rtf-and-word-docs: Addiing images to rtf...                      8      2     
utl-adding-images-to-rtf-and-word-docs: Addiing images to rtf...                      5      1     
utl-adding-images-to-rtf-and-word-docs/reportColumn.rtf at master · rogerjdea...      3      2     
utl-adding-images-to-rtf-and-word-docs/odsText.rtf at master · rogerjdeangeli...      2      2     
utl-adding-images-to-rtf-and-word-docs/reportDefine.rtf at master · rogerjdea...      2      2     
utl-adding-images-to-rtf-and-word-docs/reportSidebySide.rtf at master · roger...      2      2     
utl-adding-images-to-rtf-and-word-docs/title.rtf at master · rogerjdeangelis/...      2      2     
utl-adding-images-to-rtf-and-word-docs/imgBig.png at master · rogerjdeangeli...       2      1     
utl-adding-images-to-rtf-and-word-docs/README.md at master · rogerjdeangelis/...      2      1

**Comment by JoshMcguigan** [comment]
Which repo is it that is not shown? Can you confirm that you are seeing at least one unique visitor to that repo on github under insights -> traffic?

**Comment by rogerjdeangelis** [comment]
Hi Josh

Thanks for getting back so quickly,

I do appreciate the easy to use application. It provides me just what I
need,

I listed a repo that did not appear the output of traffic.exe.
utl_adding_images_to_rtf and docs

I expect to see at least 30 repos with new unique visitors. Last run showed
503 views and about 236 unique visttors for 14 days.

Recent top 10 ( there are a more onesies note 236 total uniques))

Probably an under count because cookies has to be enables

 REPO
REPOVIEW    REPOUSERS

 Total
 503          236

 utl-clinical-trials-futility-analysis
  43           19
 utl-shift-array-left-eliminating-missings
  38            4
 utl-excel-report-with-two-side-by-side-graphs-below_python
  32           18
 utl-safe-way-import-excel-time-value
  23           10
 utl-partitioning-your-table-for-a-big-parallel-systask-sort
  23            3
 utl-reducing-the-number-of-points-in-a-trajectory
  20            8
 utl-computing-comorbidity-indexes-using-r-and-sas-charlson
  18            9
 utl-identify-subsets-of-compound-keys-that-have-duplicates
  16            2
 utl-rearrange-the-elements-of-a-datastep-array-in-descending-order
  15            1
 utl_percentile_winsorizization_using_r_and_sas_or_wps
   9            1

**Comment by JoshMcguigan** [comment]
Ah I see it now. Thanks for bringing this up. There is a requirement that you must be the owner of the repository, which it looks like you are. But the other thing is that the github repos API by default returns only the first 30 repos (and we are requesting them sorted by most recently updated). So `traffic` needs to be updated to get the additional repos from github.

I'll work on it and update here when there is a new release. Thanks again for the feedback.

**Comment by rogerjdeangelis** [comment]
Thanks Josh

Seems to be getting harder to get info on multiple Repos.

I used to be able to get over 700 easily using python but now it fails.

Your data is accurate, you are on the right track.

I think I will investigate web scraping, python can scrape javascript
referenced pages.
It will probably be slow?

**Comment by JoshMcguigan** [comment]
I just published an update to fix this. Run `cargo install -f traffic` and let me know how it works after that.

**Comment by rogerjdeangelis** [comment]
Thanks Josh

i will fire it up in the morning.

**Comment by rogerjdeangelis** [comment]
Good Morning Josh

Getting closer.

Ran for about a minute bfore the erro.?

I can send you my token if you want to test with my repos.

C:\Windows\system32>C:\Users\beast\.cargo\bin\traffic.exe

thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value:
Error {
kind: Json(Error("missing field `uniques`", line: 4, column: 1)), url: None
}',
src\libcore\result.rs:997:5

note: Run with `RUST_BACKTRACE=1` environment variable to display a
backtrace.

C:\Windows\system32>C:\Users\beast\.cargo\bin\cargo install -f traffic

    Updating crates.io index

  Downloaded traffic v0.1.4

  Downloaded 1 crates (10.1 KB) in 0.90s

  Installing traffic v0.1.4

   Compiling winapi v0.3.6

   Compiling autocfg v0.1.2

   Compiling semver-parser v0.7.0

   Compiling rand_core v0.4.0

   Compiling winapi-build v0.1.1

   Compiling byteorder v1.3.1

   Compiling arrayvec v0.4.10

   Compiling version_check v0.1.5

   Compiling winapi v0.2.8

   Compiling libc v0.2.49

   Compiling cfg-if v0.1.6

   Compiling lazy_static v1.3.0

   Compiling stable_deref_trait v1.1.1

   Compiling nodrop v0.1.13

   Compiling scopeguard v0.3.3

   Compiling memoffset v0.2.1

   Compiling smallvec v0.6.9

   Compiling slab v0.4.2

   Compiling fnv v1.0.6

   Compiling lazycell v1.2.1

   Compiling futures v0.1.25

   Compiling siphasher v0.2.3

   Compiling proc-macro2 v0.4.27

   Compiling matches v0.1.8

   Compiling unicode-xid v0.1.0

   Compiling num-traits v0.2.6

   Compiling httparse v1.3.3

   Compiling serde v1.0.89

   Compiling lazy_static v0.2.11

   Compiling crc32fast v1.1.2

   Compiling ryu v0.2.7

   Compiling safemem v0.3.0

   Compiling try-lock v0.1.0

   Compiling scoped-tls v0.1.2

   Compiling encoding_rs v0.8.17

   Compiling percent-encoding v1.0.1

   Compiling num-integer v0.1.39

   Compiling unicode-segmentation v1.2.1

   Compiling language-tags v0.2.2

   Compiling unicode-width v0.1.5

   Compiling itoa v0.4.3

   Compiling bitflags v1.0.4

   Compiling strsim v0.7.0

   Compiling adler32 v1.0.3

   Compiling serde v0.9.15

   Compiling itoa v0.3.4

   Compiling dtoa v0.4.3

   Compiling vec_map v0.8.1

   Compiling rand_core v0.3.1

   Compiling kernel32-sys v0.2.2

   Compiling ws2_32-sys v0.2.1

   Compiling ole32-sys v0.2.0

   Compiling shell32-sys v0.1.2

   Compiling advapi32-sys v0.2.0

   Compiling unicase v1.4.2

   Compiling unicase v2.2.0

   Compiling rand_pcg v0.1.2

   Compiling rand_chacha v0.1.1

   Compiling rand v0.6.5

   Compiling semver v0.9.0

   Compiling log v0.4.6

   Compiling crossbeam-utils v0.6.5

   Compiling owning_ref v0.4.0

   Compiling unicode-normalization v0.1.8

   Compiling unicode-bidi v0.3.4

   Compiling textwrap v0.10.0

   Compiling heck v0.3.1

   Compiling rand_hc v0.1.0

   Compiling rand_xorshift v0.1.1

   Compiling rand_isaac v0.1.1

   Compiling lock_api v0.1.5

   Compiling tokio-sync v0.1.3

   Compiling relay v0.1.1

   Compiling tokio-service v0.1.0

   Compiling want v0.0.4

   Compiling rustc_version v0.2.3

   Compiling tokio-executor v0.1.6

   Compiling crossbeam-queue v0.1.2

   Compiling base64 v0.9.3

   Compiling crossbeam-epoch v0.7.1

   Compiling num_cpus v1.10.0

   Compiling parking_lot_core v0.4.0

   Compiling libflate v0.1.20

   Compiling idna v0.1.5

   Compiling num-traits v0.1.43

   Compiling quote v0.6.11

   Compiling phf_shared v0.7.24

   Compiling tokio-current-thread v0.1.5

   Compiling tokio-timer v0.2.10

   Compiling futures-cpupool v0.1.8

   Compiling mime v0.3.13

   Compiling crossbeam-deque v0.7.1

   Compiling serde_json v0.9.10

   Compiling rand_os v0.1.2

   Compiling rand_jitter v0.1.3

   Compiling net2 v0.2.33

   Compiling schannel v0.1.15

   Compiling time v0.1.42

   Compiling atty v0.2.11

   Compiling rand v0.4.6

   Compiling phf v0.7.24

   Compiling syn v0.15.27

   Compiling serde_json v1.0.39

   Compiling clap v2.32.0

   Compiling native-tls v0.1.5

   Compiling chrono v0.4.6

   Compiling uuid v0.6.5

   Compiling url v1.7.2

   Compiling tokio-threadpool v0.1.12

   Compiling phf_generator v0.7.24

   Compiling iovec v0.1.2

   Compiling parking_lot v0.7.1

   Compiling phf_codegen v0.7.24

   Compiling serde_urlencoded v0.5.4

   Compiling structopt-derive v0.2.14

   Compiling serde_derive v1.0.89

   Compiling bytes v0.4.11

   Compiling miow v0.2.1

   Compiling rpassword v2.1.0

   Compiling app_dirs v1.2.1

   Compiling mime_guess v2.0.0-alpha.6

   Compiling preferences v1.1.0

   Compiling keyring v0.6.1

   Compiling tokio-io v0.1.12

   Compiling mio v0.6.16

   Compiling structopt v0.2.14

   Compiling tokio-codec v0.1.1

   Compiling tokio-fs v0.1.6

   Compiling tokio-reactor v0.1.9

   Compiling tokio-udp v0.1.3

   Compiling tokio-tcp v0.1.3

   Compiling tokio v0.1.16

   Compiling tokio-core v0.1.17

   Compiling hyper v0.11.27

   Compiling tokio-tls v0.1.4

   Compiling hyper-tls v0.1.4

   Compiling reqwest v0.8.8

   Compiling traffic v0.1.4

    Finished release [optimized] target(s) in 2m 16s

   Replacing C:\Users\beast\.cargo\bin\traffic.exe



C:\Windows\system32>C:\Users\beast\.cargo\bin\traffic.exe

thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value:
Error {
kind: Json(Error("missing field `uniques`", line: 4, column: 1)), url: None
}',
src\libcore\result.rs:997:5

note: Run with `RUST_BACKTRACE=1` environment variable to display a
backtrace.


C:\Windows\system32>

**Comment by JoshMcguigan** [comment]
Thanks for the feedback. I just published version `0.1.5`, which improves error handling and reporting. I don't think it will entirely fix your problem, but it should give us a better idea about what the error is. It should also display the traffic data for any repos which don't have an error. 

I suspect you are getting rate limited by the github api, but let me know what output you get after running this version.

**Comment by rogerjdeangelis** [comment]
Thanks for your help

I still get the error, probably not youy problem.

I did got email about my 1000+ repos, don't know if it was from github, but
they complained that it is hard for users to find sulutions because
of the large number of repos. I sent him a 14 day summary with over 700
views and 300 unique users over the last fourteen days.
I also mentioned the github search box. I think it is easier.

The next time I tried to query with python, it did not work.

They have done something. Data on repos is githubs bread and butter and
they don't want to let users get data on thousands of repos.

Thanks again for your effort,

C:\Windows\system32>
C:\Windows\system32>C:\Users\beast\.cargo\bin\traffic.exe
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value:
Error {
kind: Json(Error("invalid type: map, expected a sequence", line: 1, column:
0)),
 url: None }', src\libcore\result.rs:997:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a
backtrace.
